### PR TITLE
Add spec for requirements of the LZMA

### PR DESCRIPTION
**Pull request by natemcmaster**
I think this comes up like every month. I wrote down my understanding of what is actually required for the fallback package cache.

### CODE REVIEW

File: docs/FallbackPackageCache.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+or %USERPROFILE%/.dotnet/sdk/NuGetFallbcakFolder, if %DOTNET_ROOT% is not writable (i.e. C:/Program Files/dotnet).
+
+## Requirements
+
+The following requirements are used to determine which packages go into the fallback package cache:
+
+ - A user should be able to restore the following templates and only use packages from the offline cache:
+    - `dotnet new console`
+    - `dotnet new library`
+    - `dotnet new web`
```
Comment (pakrym) [inline docs/FallbackPackageCache.md]: razor?

File: docs/FallbackPackageCache.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+Fallback Package Cache (LZMA)
+=============================
+
+The fallback package cache (commonly called the LZMA) is a set of NuGet packages that are bundled in the .NET Core SDK installers.
+The LZMA is a compressed file format, similar to a .zip.
+On first use, the .NET Core CLI will expand this LZMA file, extracting the packages inside to %DOTNET_ROOT%/sdk/NuGetFallbackFolder,
+or %USERPROFILE%/.dotnet/sdk/NuGetFallbcakFolder, if %DOTNET_ROOT% is not writable (i.e. C:/Program Files/dotnet).
```
Comment (livarcocc) [inline docs/FallbackPackageCache.md]: We don't do this. We always extract it next to the SDK itself.

Comment (natemcmaster) [reply docs/FallbackPackageCache.md]: Oh, what happens if C:/Program Files/dotnet isn't writable?

Comment (livarcocc) [reply docs/FallbackPackageCache.md]: You get an error saying so with a couple of alternatives on what you can do. However, usually, when you install to C:\Program Files\dotnet, that is through the MSI and the MSI expands the cache as part of the install process.

Comment (natemcmaster) [reply docs/FallbackPackageCache.md]: Cool. TIL how that works. Updated

**Pull request merged**